camayoc/tests/qpc/cli/test_scanjobs.py

```diff
@@ -217,21 +224,133 @@ def test_scanjob_with_multiple_sources(isolated_filesystem, qpc_server_config):
         assert report.get('sources', []) != []
 
 
-@pytest.mark.skip
 def test_scanjob_with_disabled_products(isolated_filesystem,
                                         qpc_server_config):
-    """Perform a scan and disable an optional product.
+    """Perform a scan with optional products disabled.
 
-    :id: b1cd9901-44eb-4e71-846c-34e1b19751d0
-    :description: Perform a scan and disable an optional product.
-    :steps: Run ``qpc scan start --sources <source> --disable-optional-products
-        <optional-product>``
+    :id: 3e01ea6c-3833-11e8-b467-0ed5f89f718b
+    :description: Perform a a scan with optional products disabled and assert
+        that the product facts are not collected in the report.
+    :steps:
+        1) Add a scan using the scan_add_and_check function
```

Comment:
maybe give fully qualified path to this function. i.e. `camayoc.tests.qpc.cli.utils.scan_add_and_check`